projects/plugins/jetpack/class.jetpack-user-agent.php

```diff
@@ -5,660 +5,7 @@
  * @package automattic/jetpack
  *
  * @deprecated 8.7.0 Use Automattic\Jetpack\Device_Detection\User_Agent_Info
- *
- * @phpcs:disable WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
- * @phpcs:disable WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
- * @phpcs:disable WordPress.NamingConventions.ValidVariableName.PropertyNotSnakeCase
- * @phpcs:disable WordPress.NamingConventions.ValidFunctionName.MethodNameInvalid
- * @phpcs:disable WordPress.Files.FileName
- */
-
-use \Automattic\Jetpack\Device_Detection\User_Agent_Info;
-
-/**
- * A class providing device properties detection.
- *
- * @deprecated 8.7.0 Use Automattic\Jetpack\Device_Detection\User_Agent_Info
  */
-class Jetpack_User_Agent_Info {
```

Comment:
This class is used in the following plugins:

- [ ] `jetpack-omnisearch`: omnisearch/class.jetpack-user-agent.php,8
- [ ] `jp-custom-css`: custom-css/class.jetpack-user-agent.php,8
- [ ] `themedy-toolbox`: custom-css/custom-css/class.jetpack-user-agent.php,8
- [ ] `wpvkp-ultimate-shortcodes`: custom-css/custom-css/class.jetpack-user-agent.php,8

Reply:
`jetpack-omnisearch` is closed, 2 JP installs. This usage is fine (it brings in the user agent the same way as the ones above, with a `jetpack_is_mobile` check... yay for code sharing). Our tool didn't find that class actually being used though. Given the small impact, going to check this box off.

The other ones were mentioned above, they included the class so we can check it off for this instance.